### PR TITLE
Add a bunch of listings for the use of mods

### PR DESCRIPTION
**Pull request by 13steinj**
I hope that I've actually fully completed this. It's complete as far as I can tell, but I need to make a few line notes.

This PR would add a /about/[locked|contests|nsfw] listing, make the current /about/unmoderated listing work for comments, and make /comments sortable. I've also included a bugfix for [this](https://www.reddit.com/r/bugs/comments/3p952j/filtered_subreddits_comments_page_is_not_respected/). You may or may not want me to add a feature flag to some of these, and you also, if you want to push b751b5f , you may want to wait until deciding on the pull request that I will make very soon after this one.

### CODE REVIEW

File: r2/r2/lib/mr_top.py [hunk 1]
```diff
@@ -134,6 +138,10 @@ def store_keys(key, maxes):
                    for item in maxes]
     query._replace(item_tuples)
 
+    if category == "sr" and thing_cls == "comment":
+        assert query2
+        query._replace(item_tuples)
```
Comment (13steinj) [inline r2/r2/lib/mr_top.py]: It might be better to just leave /r/all's `get_all_comments` method to run on demand instead of being calculated in this way.

**Comment by 13steinj** [comment]
I had to rebase in order to correct some new merge conflicts, so in case anyone is interested, [here is the previous conversation](https://github.com/13steinj/reddit/commit/c188f7fa860a30a02e58c08fb22f5f70b3747c7f#comments) with @english06

**Comment by umbrae** [comment]
Hey 13steinj, I'm so bummed to close this, but I think we're not ready to add
these extra listings. We gave it some discussion and ultimately we're just
trying to be more careful with what we add long term.

We also talked about the unmoderated queue for comments and think it'd
probably annoy a good amount of users who would really just want to use that for
links and would then have to refilter, so I think we'd like to pass on that one,
too.

Watch, though, is interesting and could fit some of these use cases. We're
going to talk about it separately a bit more and we might want to look into
that a little more. If we do, we'll either reopen this or shoot you a PM.
Sound OK?

Additionally, the /r/all-<sub>/comments commit could be worthwhile. Would you
want to open that in a separate PR?

I'm sorry that you put so much work into this and we're closing it down on
you. If you've got something that you'd like to put some serious time into,
feel free to open an issue asking if it's something we'd be interested in.

**Comment by 13steinj** [comment]
@umbrae 

It's K. I do it for fun anyway :P  If you get some kind of notification on this branch about me adding commits; I'm just pushing something to the branch that I had forgot to before; don't like to leave things unfinished 

sure thing, feel free to pm anytime (github has strangely stopped giving me around half of my notifications, so I don't trust it anymore to give me a reopening notification) :P 

I'll add the /r/all-<sub> in a separate PR :)

Thanks umbrae! Will do!

**Pull request closed without merging**